lib/glimesh_web/templates/oauth2_provider/authorization/show.html.eex

```diff
@@ -0,0 +1,5 @@
+<div class="container" style="text-align:center;">
+    <h1>Authorization code</h1>
```

Comment:
:)

Reply:
is not shown unless you navigate manually to the page as this is something that is not used *currently*